src/main/cliapp/src/components/ontologies/ZFAOntologyComponent.js

```diff
@@ -0,0 +1,85 @@
+import React, {useRef, useState} from 'react';
+import { DataTable } from 'primereact/datatable';
+import { Column } from 'primereact/column';
+import { SearchService } from '../../service/SearchService';
+import { useQuery } from 'react-query';
+import { Messages } from "primereact/messages";
+import { FilterComponent } from '../FilterComponent'
+
+import { returnSorted } from '../../utils/utils';
+
+export const ZFAOntologyComponent = () => {
+
+  const [terms, setTerms] = useState(null);
+  const [multiSortMeta, setMultiSortMeta] = useState([]);
+  const [filters, setFilters] = useState({});
+  const [page, setPage] = useState(0);
+  const [first, setFirst] = useState(0);
+  const [rows, setRows] = useState(50);
+  const [totalRecords, setTotalRecords] = useState(0);
+  const [isEnabled, setIsEnabled] = useState(true);
+  const searchService = new SearchService();
+  const errorMessage = useRef(null);
+
+  useQuery(['terms', rows, page, multiSortMeta, filters],
+    () => searchService.search('zfaTerm', rows, page, multiSortMeta, filters), {
```

Comment:
All the other endpoints this is lowercase can we make it `zfaterm` to be consistant with the rest of them?